app/subscriber/src/features/search-page/components/advanced-search/AdvancedSearch.tsx

```diff
@@ -372,7 +380,7 @@ export const AdvancedSearch: React.FC<IAdvancedSearchProps> = ({ onSearch, setSe
               <ExpandableRow icon={<FaGears />} title="Advanced Options:" hasValues={false}>
                 <Row alignItems="center" justifyContent="space-between">
                   Default operator:
-                  <FormikForm initialValues={{}} onSubmit={() => {}}>
+                  <FormikForm initialValues={initValues} onSubmit={() => {}}>
```

Comment:
The question then is why have this pointless FormikForm?

Reply:
removed FormikForm. please review again. thanks!